x-pack/plugins/timelines/public/mock/mock_timeline_data.ts

```diff
@@ -1107,6 +1107,9 @@ export const mockTimelineData: TimelineItem[] = [
               field: ['source.ip'],
               type: ['ip'],
             },
+            feed: {
+              name: ['indicator_provider'],
```

Comment:
A more realistic example here might help distinguish it from `provider` 😉

Reply:
Agree, it's value used here, I think it make more sense
<img width="1218" alt="Screenshot 2021-12-02 at 19 38 35" src="https://user-images.githubusercontent.com/7609147/144483500-66937b58-c775-4a60-be7a-95b3a2d54891.png">
re